test/jdk/tools/jpackage/share/JLinkOptionsTest.java

```diff
@@ -139,8 +141,8 @@ public void test(String javaAppDesc, String[] jpackageArgs, String[] required, S
 
     @Test
     public void testNoBindServicesByDefault() {
-        final var defaultModules = getModulesInRuntime();
-        final var modulesWithBindServices = getModulesInRuntime("--bind-services");
+        final var defaultModules = getModulesInRuntime("--limit-modules java.smartcardio,jdk.crypto.cryptoki,java.desktop");
+        final var modulesWithBindServices = getModulesInRuntime("--bind-services --limit-modules java.smartcardio,jdk.crypto.cryptoki,java.desktop");
```

Comment:
Pre-existing: It would be good to document which additional modules in the set get dragged in by `--bind-services`. That would allow future readers of that test to figure out why a test starts to fail (e.g. by a future change removing a provided service in a module).

Reply:
This is the diff of modules between "jpackage without --bind-services" and "jpackage with --bind-services" before this PR:
```
[20:27:10.849] TRACE: assertNotEquals(, jdk.charsets,jdk.crypto.cryptoki,jdk.crypto.mscapi,jdk.editpad,jdk.jdeps,jdk.jlink,jdk.jpackage,jdk.jstatd,jdk.localedata,jdk.naming.dns,jdk.naming.rmi): Check '--bind-services' option adds modules
```

Reply:
I added a comment about what is expected.